wiki/Linux-Troubleshooting.md

```diff
@@ -11,34 +11,47 @@ This path might differ based on your Steam installation, in that case SteamVR wi
 Then pick the storage location with the star emoji (⭐) and take the path directly above the usage statistics. Prepend this path to `steamapps/common/SteamVR/bin/vrmonitor.sh`.
 Finally put this entire path into the SteamVR commandline options instead of the other one.
 
-### Hyprland/Sway/Wlroots Fix
+### Hyprland/Sway/Wlroots Qt fix
 
 If you're on hyprland, sway, or other wlroots-based wayland compositor, you might have to prepend `QT_QPA_PLATFORM=xcb` before commandline, which results in full commandline for steamvr being something like this:
 `QT_QPA_PLATFORM=xcb ~/.local/share/Steam/steamapps/common/SteamVR/bin/vrmonitor.sh %command%`.
 
 Related issue:
 [[BUG] No SteamVR UI on wlroots-based wayland compositors (sway, hyprland, ...) with workaround](https://github.com/ValveSoftware/SteamVR-for-Linux/issues/637).
 
+
+## The alvr driver doesn't get detected by SteamVR (even after vrmonitor fix)
+
+Could be related to Arch AUR package (either installed not for nvidia on nvidia based system (`alvr-nvidia`), or just in general).
+
+### Fix
+
+Try using a launcher or portable .tar.gz release from the Releases page.
+
 ## Artifacting, no SteamVR Overlay or graphical glitches in streaming view
 
-Could be related to AMD amdvlk driver being present on your system.
+Could be related to AMD amdvlk or amdgpu-pro driver being present on your system.
 
 If you have Amdvlk installed on your system, it overrides other vulkan drivers and causes SteamVR to break. Use the `vulkan-radeon` driver (aka radv) instead.
 
 ### Fix
 
-Check if Amdvlk is installed by seeing if `ls /usr/share/vulkan/icd.d/ | grep amd_icd` shows anything. If so, uninstall Amdvlk from your system.
+Check if amdvlk or other vulkan drivers are installed by seeing if `ls /usr/share/vulkan/icd.d/ | grep amd_icd` shows anything. If so, uninstall amdvlk from your system.
+
+On arch, first install `vulkan-radeon` and uninstall other drivers.
 
-## Failed to create VAAPI encoder (fedora)
+## Failed to create VAAPI encoder
 
 Blocky or crashing streams of gameplay and then an error window on your desktop saying:
 > Failed to create VAAPI encoder: Cannot open video encoder codec: Function not implemented. Please make sure you have installed VAAPI runtime.
 
-This seems to be an issue for AMD GPU fedora 39+ users, but maybe others.
-
```

Comment:
it isn't exhaustive but it's like 3 distros or something and we should at least mention them
and potentially include fixes or provide a general explanation on what to look for to fix it